nvflare/lighter/impl/aws_template.yml

```diff
@@ -12,14 +12,21 @@ aws_start_sh: |
   check_binary dig "Please install it first."
   check_binary jq "Please install it first."
 
+  if [ -z ${vpc_id+x} ]
+  then
+      using_default_vpc=true
+  else
+      using_default_vpc=false
+  fi
+
   if [ -z ${image_name+x} ]
   then
       container=false
   else
       container=true
   fi
 
-  if [ $container = true ]
+  if [ $container == "true" ]
```

Comment:
why is the changing from "=" to "=="?

Reply:
I tried to make all comparison to ``==`` as bash complains unary operator.